app/src/main/res/layout/category_row_item.xml

```diff
@@ -1,35 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.CardView
+<android.support.design.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/scoring_category_card_view"
-    android:layout_width="@dimen/category_card_width"
+    xmlns:card_view="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
     android:layout_height="@dimen/category_card_height"
-    app:cardCornerRadius="@dimen/cardview_default_radius"
-    app:cardElevation="@dimen/cardview_default_elevation"
     >
 
-    <LinearLayout
+    <android.support.v7.widget.CardView
+        android:id="@+id/scoring_category_card_view"
+        style="@style/AppTheme.IconButtonCardView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="vertical"
+        android:layout_height="@dimen/category_card_height"
+        android:layout_gravity="center"
+        android:layout_marginBottom="@dimen/activity_vertical_margin"
+        android:layout_marginTop="@dimen/activity_vertical_margin"
+        android:background="#fdfdfd"
+        android:padding="@dimen/activity_vertical_margin"
         >
 
-        <TextView
-            android:id="@+id/category_item_title"
-            style="@style/AvenirNext_DemiBold"
+        <RelativeLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:textColor="@color/colorAccent"
-            android:textSize="@dimen/category_title_text_size"
-            />
+            android:layout_height="match_parent"
+            android:layout_marginLeft="@dimen/activity_vertical_margin"
+            >
 
-        <TextView
-            android:id="@+id/category_item_description"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            />
+            <TextView
+                android:id="@+id/category_item_title"
+                style="@style/AvenirNext_DemiBold"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="2dp"
```

Comment:
This is a very small margin, you probably don't need it. If you do, make it a dimension variable